nmdc_runtime/site/repository.py

```diff
@@ -852,6 +853,69 @@ def biosample_submission_ingest():
     ]
 
 
+@repository
+def biosample_export():
+    normal_resources = run_config_frozen__normal_env["resources"]
+    return [
+        nmdc_study_to_ncbi_submission_export.to_job(
+            resource_defs=resource_defs,
+            config={
+                "resources": merge(
+                    unfreeze(normal_resources),
+                    {
+                        "mongo": {
+                            "config": {
+                                "host": {"env": "MONGO_HOST"},
+                                "username": {"env": "MONGO_USERNAME"},
+                                "password": {"env": "MONGO_PASSWORD"},
+                                "dbname": {"env": "MONGO_DBNAME"},
+                            },
+                        },
+                        "runtime_api_site_client": {
+                            "config": {
+                                "base_url": {"env": "API_HOST"},
+                                "client_id": {"env": "API_SITE_CLIENT_ID"},
+                                "client_secret": {"env": "API_SITE_CLIENT_SECRET"},
+                                "site_id": {"env": "API_SITE_ID"},
+                            },
+                        },
+                    },
+                ),
+                "ops": {
+                    "get_ncbi_export_pipeline_study_id": {
+                        "config": {
+                            "nmdc_study_id": "",
+                        }
+                    },
+                    "get_ncbi_export_pipeline_inputs": {
+                        "config": {
+                            "nmdc_ncbi_attribute_mapping_file_url": "",
+                            "ncbi_submission_metadata": {
+                                "email": "",
+                                "first": "",
+                                "last": "",
+                                "user": "",
+                                "organization": "",
+                            },
+                            "ncbi_bioproject_metadata": {
```

Comment:
Code should be able to get this from OmicsProcessing

Reply:
It could also be a slot on `Study`

Reply:
Addressed this.